portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,101 @@
+// // Copyright 2019 Google LLC
```

Comment:
Can we leave this file out since it's unused?

Reply:
I am not sure if I am allowed to take out licensing but I suppose if you think so then I probably can?

Reply:
No I mean the entire file (and no please don't remove the licensing).